x-pack/solutions/observability/plugins/observability/public/pages/alert_details/alert_details.test.tsx

```diff
@@ -61,6 +62,7 @@ const mockKibana = () => {
         basePath: {
           prepend: jest.fn(),
         },
+        get: jest.fn().mockReturnValue({ alertContext: [] }),
```

Comment:
To avoid the following error:

![image](https://github.com/user-attachments/assets/cb570809-a38e-458a-9b6e-8629c6be04fc)